src/LayoutManager/README.md

```diff
@@ -2,6 +2,39 @@
 
 Layout manager exposes a main public method that performs group layout given a target group and target objects.
 
+## Expectations for non interactive groups
+
+The layout manager will calculate the size and position of the group when is created or updated
+The FitContent strategy is the classic fabric 5.x group and will grow or shrink when objects are added or removed.
+In general the expectation is that creating or modifying an object will create a group with a postion and once that group is added to canvas, the position of the objects will be the same visually that those object would have had on the canvas if they weren't on the group.
```

Comment:
This is correct only if left and top are not passed
Could it be you missed that?

Reply:
is written below indeed in the paragraph about position.

Reply:
confusing